connection.js

```diff
@@ -691,8 +691,8 @@ function getArgNameStringsGRU(familyNameString) {
 
 function rankToString(r) {
     const R = Math.floor(r);
-    if (R >= 30)  return `${R - 30 + 1}d`; // R >= 30: 1 dan or stronger
```

Comment:
This was clearer with the `:` in i think.

Reply:
ok, i wanted to remove as much punctuation as possible like markdown lint does, but it is indeed clearer in that case :1st_place_medal:

Reply:
fixed